Address asciidoctor-dita-vale issues for managing packages (#4364)
theforeman/foreman-documentation: guides/common/modules/proc_managing-packages-from-a-host-by-using-api.adoc

:_mod-docs-content-type: PROCEDURE

[id="managing-packages-from-a-host-by-using-api"]

> = Managing packages from a host by using API

Reviewer: ```suggestion
= Managing packages on a host by using API
```

[role="_abstract"]
You can install, upgrade, or remove packages from a host by using {Project} API.

> This procedure uses the `katello_package_install` to install a package.

Reviewer: ```suggestion
The example below uses the `katello_package_install` action to install a package.
```

.Procedure
. Specify the API request body in the JSON format.
For example:
+
[options="nowrap", json, subs="+quotes,verbatim,attributes"]
----
{
  "job_invocation" : {
    "concurrency_control" : {
      "concurrency_level" : 100
    },
    "feature" : "katello_package_install",
    "inputs" : {
      "package" : "nano vim"
    },
    "scheduling" : {
      "start_at" : "2023-09-21T19:00:00+00:00",
      "start_before" : "2023-09-23T00:00:00+00:00"
    },
    "search_query" : "*",
    "ssh" : {
      "effective_user" : "My_Username",
      "effective_user_password" : "My_Password"
    },
    "targeting_type" : "dynamic_query"
  }
}
----
The example uses the following notable values:
+
"job_invocation":: Specifies the object which contains the API request.
"concurrency_level":: Optional. Limits the number of hosts on which the job is run concurrently.
"feature":: Specifies the action to be performed.
Use `"katello_package_install"` to install a package, `"katello_package_upgrade"` to upgrade a package, or `"katello_package_remove"` to remove a package.
"package":: Specifies the packages on which the action is performed.
Separate multiple packages with a whitespace.

> "scheduling":: Optional. Specifies time boundaries for when to start to package action.

Reviewer: ```suggestion
"scheduling":: Optional. Specifies time boundaries for when to start the package action.
```
